lang/java/avro/pom.xml

```diff
@@ -164,6 +164,11 @@
       <artifactId>xz</artifactId>
       <version>${tukaani.version}</version>
     </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <optional>true</optional>
```

Comment:
I think optional is correct. According to the [optional maven doc](https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html):

> If a user wants to use functionality related to an optional dependency, they will have to redeclare that optional dependency in their own project.